### DISCUSSION TITLE
Control AdGuard Home with a Browser extension

### DISCUSSION THREAD

**Discussion by screendriver** [AdguardTeam/AdGuardHome#4311] [Ideas]
It would be great if we could enable or disable AdGuard Home with a browser extension like others [already can do](https://chrome.google.com/webstore/detail/pihole-browser-extension/ngoafjpapneaopfkpboebcahajopcifi?hl=de) with a `Pi-hole` installation. With that I don't have to open the AdGuard Home website all the time to temporarily disable it.

I already wrote an email to the contact persons of the [AdGuard Chrome Extension](https://chrome.google.com/webstore/detail/adguard-adblocker/bgnkhhnnamicmpeenaelnjfhikgbkllg?hl=de) but they told me that I should write my request here.

**Comment by samcro1967** [comment]
Also interested in this.

**Comment by Javican** [comment]
I have been lokking for something like this for a long time, since I moved fro Pihole I miss the browser functionality to whitelist, blacklist a site from the browser.
Also it would be great to have the functionality to temporarily disable adguard home for 1-5-15 minutes instead of disabling and enabling again

**Reply by Javican** [reply]
Do we know if Adguard home has API functionality since without that we are almost dead

**Reply by MC-r15ti4n** [reply]
I would even go a step further. Since I have different clients with different users, it would be good if you could temporarily turn off the protection for each client instead of globally. Also, this should not be done with the AdGuard Admin's credentials.

**Comment by samcro1967** [comment]
Sort of...[Link](https://github.com/AdguardTeam/AdGuardHome/blob/master/openapi/openapi.yaml)

**Comment by yasars** [comment]
no idea?

**Comment by stephan1827** [comment]
Any news on this?

**Comment by lostcowboy** [comment]
I would like to see this from AdGuard but there is a Browser extension that will disable AdGuard Home [Adguard Home Browser Extension](https://github.com/satheshshiva/Adguard-Home-Browser-Ext)

**Comment by stevenrt** [comment]
I really want this feature (control of AdGuard Home from a browser extension).  I'm considering moving to Pi-Hole because of this.

**Reply by balkerrman** [reply]
> I really want this feature (control of AdGuard Home from a browser extension). I'm considering moving to Pi-Hole because of this.

I use this, works great: https://chromewebstore.google.com/detail/adguard-home-control/flbpjbdbniigclhibiemmeopcnfimajj?pli=1
